common/djangoapps/student/views.py

```diff
@@ -340,7 +340,7 @@ def _cert_info(user, course_overview, cert_status, course_mode):  # pylint: disa
             if course_overview.has_any_active_web_certificate:
```

Comment:
I think this should stay, actually -- it's not known that get_certificate_url also makes a similar check at this point in the workflow, and the outer check prevents any of the inner work from taking place at all.  From a performance perspective, if we do end up executing the check a second time, we'll reuse the initialized+cached objects versus hitting the backend services again.  I could be wrong, tho?